CSIP17 - testCase.xml and 3 corpora packages

PhillipTommerholt opened the pull request:
Updated with testCase.xml, real testCase.xsd (no symlink) and corpora packages

PhillipTommerholt committed:
Updated with testCase.xml and corpora packages

Updated with testCase.xml and corpora packages

carlwilson committed:
FIX - merge conflicts.

carlwilson commented:
Conflicts fixed ready for merging.

carlwilson reviewed (changes requested):
Fix package paths in testCase.xml document

PhillipTommerholt committed:
Update testCase.xml

Updated testCase.xml to fix paths and change isValid values

PhillipTommerholt committed:
Updated type values in METS

Updated type values in METS in 3 IPs

andersbonielsen reviewed (dismissed):
Value updated.

PhillipTommerholt committed:
updated csip17_1

updated csip17_1

PhillipTommerholt committed:
updated to cause less errors

the IP still has other flaws like MD5 fixity errors

PhillipTommerholt committed:
csip17_2 and csip17_3

Updated csip17_2 and csip17_3

PhillipTommerholt committed:
Update METS.xml

updated METS to the correct profile https://earkcsip.dilcis.eu/profile/E-ARK-CSIP.xml

ltaimre committed:
Merge remote-tracking branch 'origin/integration' into csip/dmdSec/CSIP17

carlwilson committed:
Merge branch 'integration' into csip/dmdSec/CSIP17
